ui-admin/src/portal/dashboard/widgets/WebsiteWidget.tsx

```diff
@@ -0,0 +1,102 @@
+
+export const WebsiteWidget = ({ portal }: { portal: Portal }) => {
+  const livePortalUrl = portal.portalEnvironments.find(env =>
+    env.environmentName === 'live')?.portalEnvironmentConfig.participantHostname
+
+  return (
+    <InfoCard>
+      <InfoCardHeader>
+        <div className="d-flex align-items-center justify-content-between w-100">
+          <span className="fw-bold">Website</span>
+          <div className="d-flex">
+            <Button onClick={() => {
+              const url = livePortalUrl?.startsWith('http') ? livePortalUrl : `https://${livePortalUrl}`
+              window.open(url, '_blank')
+            }}
+            tooltip={'Preview live website'}
+            variant="light" className="border m-1">
+              <FontAwesomeIcon icon={faEye} className="fa-lg"/> Preview
+            </Button>
+            <CustomizeWebsiteDropdown portal={portal}/>
+          </div>
+        </div>
+      </InfoCardHeader>
+      <InfoCardBody>
+        <div className="container">
+          <div className="w-100">
+            <div style={{ minHeight: '200px' }} className="d-flex justify-content-center align-items-center">
+              {livePortalUrl ?
+                <Button
+                  variant="light"
+                  className="border"
+                  onClick={() => {
+                    const url = livePortalUrl?.startsWith('http') ? livePortalUrl : `https://${livePortalUrl}`
+                    window.open(url, '_blank')
+                  }}
+                >
+                  <FontAwesomeIcon icon={faArrowUpRightFromSquare}/> Visit website
+                </Button> :
+                <span className="text-muted bg-gray">Website not yet published</span>
+              }
+            </div>
+          </div>
+        </div>
+      </InfoCardBody>
+    </InfoCard>
+  )
+}
+
+const CustomizeWebsiteDropdown = ({ portal }: { portal: Portal }) => {
+  const navigate = useNavigate()
+
+  return (
+    <div className="dropdown">
+      <Button
+        data-bs-toggle='dropdown'
+        className="dropdown-toggle border m-1"
+        type="button"
+        id="customizeWebsiteMenu"
+        variant="light"
+        aria-haspopup="true"
+        aria-expanded="false"
+        aria-label={'Customize website'}
+      >
+        <FontAwesomeIcon icon={faGear} className="fa-lg"/> Customize
+      </Button>
+      <div className="dropdown-menu" aria-labelledby="customizeWebsiteMenu">
+        <DropdownButton
+          onClick={() => navigate(siteContentPath(portal.shortcode, 'sandbox'))}
+          icon={faPencil}
+          label="Edit website"
+          description="Design your portal website"
+        />
+        <div className="dropdown-divider my-1"></div>
+        {/* rendering the existing randing modal within the context of the portal dashboard
+            isn't particularly clean, from a user experience perspective
+            for now, we'll direct users to the sandbox site editor to edit branding */}
```

Comment:
I found this confusing - I vote on just not having this as a button. Or maybe the above one should be `Edit website & branding`